pkgs/tools/compression/brotli/default.nix

```diff
@@ -15,11 +15,12 @@ stdenv.mkDerivation rec {
 
   nativeBuildInputs = [ cmake ];
 
-  patches = stdenv.lib.optional staticOnly (fetchpatch {
-    # from https://github.com/google/brotli/pull/655
-    url = "https://github.com/google/brotli/commit/7289e5a378ba13801996a84d89d8fe95c3fc4c11.patch";
-    sha256 = "1bghbdvj24jrvb0sqfdif9vwg7wx6pn8dvl6flkrcjkhpj0gi0jg";
-  });
+  patches = stdenv.lib.optional staticOnly ./static.patch;
+  #(fetchpatch {
+  #  # from https://github.com/google/brotli/pull/655
+  #  url = "https://github.com/google/brotli/commit/7289e5a378ba13801996a84d89d8fe95c3fc4c11.patch";
+  #  sha256 = "1bghbdvj24jrvb0sqfdif9vwg7wx6pn8dvl6flkrcjkhpj0gi0jg";
+  #});
```

Comment:
What was the fetch issue? I see you opted to commit the patch.

Reply:
The patch was an not mergeable with their master/release branch, as another PR doing the things differently was merged, and the patch was not up to date.

I thought it would be easier to keep the modification here

Reply:
Btw didn't meant to keep it as a comment, I can remove it

Reply:
Ah, then I suggest you send that new patch as a PR upstream and then grab it as the previous one.

`url = "https://github.com/google/brotli/commit/[commit hash of new patch here].patch";`

Reply:
> Btw didn't meant to keep it as a comment, I can remove it

You generally don't comment out things that don't work in git. You can fix it, remove it or comment it our with a comment why it is commented and a TODO.

Reply:
Done, but apparenty their CI doesn't like the patch, anyway on our side, it works for both static and shared libraries